### PR TITLE
ensure em_asm template get C++ linkage ( cmake builds )

### PR DESCRIPTION
**Pull request by pmp-p**
to avoid on recent emsdk (clang upstream)
```
In file included from /opt/python-wasm-sdk/emsdk/upstream/emscripten/cache/sysroot/include/emscripten/emscripten.h:23:
/opt/python-wasm-sdk/emsdk/upstream/emscripten/cache/sysroot/include/emscripten/em_asm.h:151:1: error: templates must have C++ linkage
  151 | template<typename, typename = void> struct __em_asm_sig {};
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### CODE REVIEW

**Comment by pmp-p** [comment]
got no problem with build_web.sh

File: include/pocketpy/pocketpy_c.h
Comment (blueloveTH) [inline include/pocketpy/pocketpy_c.h:1]: remove the change in this file. I think `build_web.sh` does not use `pocketpy_c.h`.

Comment (pmp-p) [reply include/pocketpy/pocketpy_c.h:1]: i think build static libpocketpy.a with cmake does

**Comment by blueloveTH** [comment]
```cpp
    // #include <emscripten.h> remove this
    // leave the following empty
    #define PK_EXPORT
```

**Comment by pmp-p** [comment]
but you need EMSCRIPTEN_KEEPALIVE no ?

**Comment by blueloveTH** [comment]
> but you need EMSCRIPTEN_KEEPALIVE no ?

Could we just copy the macro definition instead of using the whole header?

**Comment by blueloveTH** [comment]
Please do not change line endings.
image

**Pull request merged**